pkgs/development/python-modules/trezor/default.nix

```
@@ -33,7 +33,7 @@ buildPythonPackage rec {
   # disable test_tx_api.py as it requires being online
   checkPhase = ''
     runHook preCheck
-    ${python.interpreter} -m pytest --pyargs trezorlib.tests.unit_tests --ignore trezorlib/tests/unit_tests/test_tx_api.py
+    ${python.interpreter} -m pytest --pyargs tests --ignore tests/test_tx_api.py
```

if you want to, doesn't really matter :)
```suggestion
     pytest --pyargs tests --ignore tests/test_tx_api.py
```

---

Thanks!